src/providers/sh/commands/alias/get-app-name.js

```diff
@@ -14,9 +14,11 @@ async function getAppName(output: Output, localConfig?: string) {
   }
 
   // Otherwise try to get it from the package
-  const pkg = await readPackage()
-  if (!(pkg instanceof NowError) && pkg) {
-    return pkg.name
+  if (!(config instanceof NowError) && config.type === "npm") {
```

Comment:
`!config.type || config.type === "npm"` since it could still be `npm` even if we don't have a `type`